FrontEnd/src/components/ProfileDetail/MainInfo/ProfileDetailNavBar.jsx

```diff
@@ -10,38 +12,65 @@ const MENU_LINKS = {
     'cooperation': 'Формат співпраці',
 };
 
-function ProfileDetailNavBar() {
-
+function ProfileDetailNavBar({ data }) {
   const { hash } = useLocation ();
+  const navigate = useNavigate ();
+  const [activeLink, setActiveLink] = useState('');
+
+  useEffect(() => {
+    if (hash) {
+      setActiveLink(hash.substring(1));
+    } else {
+      setActiveLink(
+        data.is_registered && document.getElementById('about-company') ? 'about-company' :
+        data.is_startup && document.getElementById('startup') ? 'startup' : ''
```

Comment:
This is a very bad practice to rely on DOM. Do we need to use `document.getElementById` here and could we replace it?

Reply:
Yes, I read about it. But I didn't find another solution. 
Only tabs for  components with provided information have to be shown. And I wanted to check if component is on the page using id of that component. 
Maybe you can recommend some solution? What is better to use here?

Reply:
Menu tabs functionality has been implemented with Context.